framework/src/controller/schema/application.js

```diff
@@ -13,6 +13,211 @@ module.exports = {
+			},
+			BLOCK_RECEIPT_TIMEOUT: {
+				type: 'integer',
+			},
+			EPOCH_TIME: {
+				type: 'object',
+			},
+			FEES: {
+				type: 'object',
+				$ref: '#/fees',
+			},
+			MAX_PAYLOAD_LENGTH: {
+				type: 'integer',
+			},
+			MAX_PEERS: {
+				type: 'integer',
+			},
+			MAX_SHARED_TRANSACTIONS: {
+				type: 'integer',
+			},
+			MAX_TRANSACTIONS_PER_BLOCK: {
+				type: 'integer',
+			},
+			MAX_VOTES_PER_TRANSACTION: {
+				type: 'integer',
+			},
+			MAX_VOTES_PER_ACCOUNT: {
+				type: 'integer',
+			},
+			MIN_BROADHASH_CONSENSUS: {
+				type: 'integer',
+			},
+			MULTISIG_CONSTRAINTS: {
+				type: 'object',
+				required: ['MIN', 'LIFETIME', 'KEYSGROUP'],
+				properties: {
+					MIN: {
+						type: 'object',
+						required: ['MINIMUM', 'MAXIMUM'],
+						properties: {
+							MINIMUM: {
+								type: 'integer',
+							},
+							MAXIMUM: {
+								type: 'integer',
+							},
+						},
+					},
+					LIFETIME: {
+						type: 'object',
+						required: ['MINIMUM', 'MAXIMUM'],
+						properties: {
+							MINIMUM: {
+								type: 'integer',
+							},
+							MAXIMUM: {
+								type: 'integer',
+							},
+						},
+					},
+					KEYSGROUP: {
+						type: 'object',
+						required: ['MIN_ITEMS', 'MAX_ITEMS'],
+						properties: {
+							MIN_ITEMS: {
+								type: 'integer',
+							},
+							MAX_ITEMS: {
+								type: 'integer',
+							},
+						},
+					},
+				},
+			},
+			NETHASHES: {
+				type: 'array',
+				items: {
+					type: 'string',
+					pattern: '^[a-fA-F0-9]{64}$',
+				},
+			},
+			NORMALIZER: {
+				type: 'string',
+				pattern: '^[0-9]+$',
+			},
+			REWARDS: {
+				type: 'object',
+				required: ['MILESTONES', 'OFFSET', 'DISTANCE'],
+				properties: {
+					MILESTONES: {
+						type: 'array',
+						items: {
+							type: 'string',
+							pattern: '^[0-9]+$',
+						},
+					},
+					OFFSET: {
+						type: 'integer',
+					},
+					DISTANCE: {
+						type: 'integer',
+					},
+				},
+			},
+			TOTAL_AMOUNT: {
+				type: 'string',
+				pattern: '^[0-9]+$',
+			},
+			UNCONFIRMED_TRANSACTION_TIMEOUT: {
+				type: 'integer',
+			},
+			EXPIRY_INTERVAL: {
+				type: 'integer',
+			},
+		},
+	},
+
+	fees: {
+		$id: '#/fees',
+		type: 'object',
+		required: [
+			'SEND',
+			'VOTE',
+			'SECOND_SIGNATURE',
+			'DELEGATE',
+			'MULTISIGNATURE',
+			'DAPP_REGISTRATION',
+			'DAPP_WITHDRAWAL',
+			'DAPP_DEPOSIT',
+		],
+		properties: {
+			SEND: {
+				type: 'string',
+				pattern: '^[0-9]+$',
+			},
+			VOTE: {
+				type: 'string',
+				pattern: '^[0-9]+$',
+			},
+			SECOND_SIGNATURE: {
+				type: 'string',
+				pattern: '^[0-9]+$',
+			},
+			DELEGATE: {
+				type: 'string',
+				pattern: '^[0-9]+$',
+			},
+			MULTISIGNATURE: {
+				type: 'string',
+				pattern: '^[0-9]+$',
+			},
+			DAPP_REGISTRATION: {
+				type: 'string',
+				pattern: '^[0-9]+$',
+			},
+			DAPP_WITHDRAWAL: {
+				type: 'string',
+				pattern: '^[0-9]+$',
+			},
+			DAPP_DEPOSIT: {
+				type: 'string',
+				pattern: '^[0-9]+$',
+			},
```

Comment:
Please add a string format `bignumber` and then set format everywhere in schema. Don't use redundant patterns.